code/_onclick/hud/screen_alarms.dm

```diff
@@ -160,9 +174,101 @@ var/global/list/screen_alarms_locs = list(
 
 /obj/abstract/screen/alert/object/buckled
 	name = "Buckled"
-	desc = "You've been buckled to something and can't move. Click this alert to unbuckle unless you're unable to."
+	desc = "You've been buckled to something and can't move. Click on this alert to unbuckle."
+
+//Carbon Alarms
+/obj/abstract/screen/alert/carbon/breath
+	name = "Choking"
```

Comment:
technically, this is really suffocating, choking implies something obstructing the airway